src/app/space/add-app-overlay/add-app-overlay.component.ts

```diff
@@ -23,14 +23,15 @@ import { Application, DeploymentApiService } from '../create/deployments/service
 })
 export class AddAppOverlayComponent implements OnDestroy, OnInit {
   currentSpace: Space;
-  isProjectNameValid: boolean;
+  isProjectNameValid = false;
   loggedInUser: User;
   projectName: string = '';
   selectedFlow: string = '';
   spaces: Space[] = [];
   subscriptions: Subscription[] = [];
   applications: string[] = [];
-  isProjectNameAvailable: boolean;
+  isProjectNameAvailable = false;
```

Comment:
needs a type

Reply:
when we assign a primitive value, type is not needed by default. Type is defined from the primitive value we assign